lib/css-plugin/index.js

```diff
@@ -0,0 +1,61 @@
+const postcss = require('postcss');
+const concat = require('broccoli-concat');
+const Filter = require('broccoli-persistent-filter');
+
+CSStoJSON.prototype = Object.create(Filter.prototype);
+CSStoJSON.prototype.constructor = CSStoJSON;
+CSStoJSON.prototype.extensions = ['js'];
+CSStoJSON.prototype.targetExtension = 'js';
+
+function CSStoJSON(inputTree, options = {}) {
+  if (!(this instanceof CSStoJSON)) {
+    return new CSStoJSON(inputTree, options);
+  }
+
+  let mergedTree = concat(inputTree, {
+    outputFile: options.outputFile,
+    sourceMapConfig: { enabled: false },
+    allowNone: true
+  });
+
+  Filter.call(this, mergedTree, {
+    annotation: options.annotation
+  });
+}
+
+CSStoJSON.prototype.processString = function(content) {
+  return processConfigFile(content);
+};
+
+function processConfigFile(str) {
```

Comment:
This processor uses POST CSS to take any of the files that have the `.css-compose` extension and converts it into a file that the ember app can consume. So, at its simplest, this:

```css
@composer {
  @a {
    composes: 'b', 'c';
  }
}
```

...would become this:

```js
export default {
  'a': ['b', 'c']
};
```

There's a few choices here that are easily up for discussion (might open issues for them). 

1. Is using the `.css-compose` the right choice? Or should we just use the existing `.css` files? If we went the latter route, that would mean you could theoretically also add additional styles in there. But, this is also not valid CSS so we might cause some issues with preprocessors and such. 

2. Is using the `@composer` necessary? Or, should it just be something like:

```css
@a {
  composes: 'b', 'c';
}
```

The problem with this is that if you had a class name called "media", then we're colliding with actual CSS.